### PR TITLE
Menu changes

### PR DESCRIPTION
**Pull request by jamieolivier**

### CODE REVIEW

File: app/_includes/header.html [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+    <a id="logo" class="logo white" href="/">
       <img src="/img/logo_w.png" alt="silicon rhino logo"
     /></a>
     <button id="menu" class="">
       <span>toggle menu</span>
     </button>
-    <ul class="main-links">
-      <li class="no"><a class="cta" href="/contact/">Get A Quote</a></li>
+  </div>
+<div class="main-menu">
```
Comment (ray-silicon-rhino) [inline app/_includes/header.html:10]: indents?

File: app/_scss/main.scss [hunk 2]
```diff
@@ -257,7 +257,10 @@ a {
     display: none;
   }
   #menu {
-    float: right;
+    top: 10px;
+    right: 10px;
+    position: fixed;
+    z-index: 100000000;
```
Comment (ray-silicon-rhino) [inline app/_scss/main.scss:263]: is this necessary? is a bit big. :-D

File: app/_scss/main.scss [hunk 3]
```diff
@@ -312,14 +315,11 @@ a {
   display: block;
   $menu-width: 100%;
   > div {
-    background-color: rgba(0, 129, 144, 0.8);
+    background-color: rgba(0, 129, 144, 1);
```
Comment (ray-silicon-rhino) [inline app/_scss/main.scss:318]: this can be just rgb

File: app/_scss/main.scss [hunk 4]
```diff
@@ -368,8 +366,193 @@ a {
...
+}
+#main-menu {
+  position: absolute;
+  top: 5px;
+  right: 10px;
+}
+.main-menu {
+  display: block;
+  > .cta {
+    position: fixed;
```
Comment (ray-silicon-rhino) [inline app/_scss/main.scss:380]: 2 positions

File: app/_scss/main.scss [hunk 5]
```diff
@@ -368,8 +366,193 @@ a {
...
+    position: absolute;
+    top: 18px;
+    right: 10%;
+  }
+  > div {
+    height: 100%;
+    background-color: rgba(0, 129, 144, 1);
```
Comment (ray-silicon-rhino) [inline app/_scss/main.scss:387]: this can be just rgb

File: app/_scss/main.scss [hunk 6]
```diff
@@ -368,8 +366,193 @@ a {
...
+}
+
+.drawer.active {
+  top: 0;
+  -webkit-transform: translate3d(0, 0, 0);
+  -moz-transform: translate3d(0, 0, 0);
+  -o-transform: translate3d(0, 0, 0);
+  transform: translate3d(0, 0, 0);
+}
+.main-menu > div > ul {
```
Comment (ray-silicon-rhino) [inline app/_scss/main.scss:422]: maybe is better to assign classes?

File: app/_scss/main.scss [hunk 7]
```diff
@@ -368,8 +366,193 @@ a {
...
+  font-size: 30px;
+}
+#main-menu {
+  position: absolute;
+  top: 5px;
+  right: 10px;
+}
+.main-menu {
+  display: block;
+  > .cta {
```
Comment (ray-silicon-rhino) [inline app/_scss/main.scss:379]: maybe is better to assign an specific class?

File: app/_scss/main.scss [hunk 8]
```diff
@@ -368,8 +366,193 @@ a {
...
+    }
+    > a:after {
+      background: none repeat scroll 0 0 transparent;
+      bottom: 0;
+      content: "";
+      display: block;
+      height: 2px;
+      left: 50%;
+      position: absolute;
+      background: $darkblue;
```
Comment (ray-silicon-rhino) [inline app/_scss/main.scss:449]: there are 2 background properties

File: app/_scss/main.scss [hunk 9]
```diff
@@ -368,8 +366,193 @@ a {
...
+      }
+      &:nth-child(2) {
+        transition-delay: 0.625s;
+      }
+      &:nth-child(3) {
+        transition-delay: 0.75s;
+      }
+    }
+  }
+  #cross {
```
Comment (ray-silicon-rhino) [inline app/_scss/main.scss:492]: maybe is better to use a class here?

File: app/_scss/main.scss [hunk 10]
```diff
@@ -368,8 +366,193 @@ a {
...
+    span {
+      display: block;
+      background: white;
+      width: 50px;
+      height: 6px;
+      position: relative;
+      top: 24px;
+      left: 20px;
+      margin: 10px 0;
+      &:nth-child(1) {
```
Comment (ray-silicon-rhino) [inline app/_scss/main.scss:481]: maybe is better to use classes here? as it'd be easier to maintain in the future

File: app/_scss/main.scss [hunk 11]
```diff
@@ -368,8 +366,193 @@ a {
...
+        transition-delay: 0.125s;
+      }
+      &:nth-child(3) {
+        transition-delay: 0.25s;
+      }
+    }
+  }
+  #cross {
+    span {
+      &:nth-child(1) {
```
Comment (ray-silicon-rhino) [inline app/_scss/main.scss:539]: maybe is better to use classes on these spans?

**Review by ray-silicon-rhino — APPROVED** [review]

**Review by joubertasr — APPROVED** [review]
Looks good, just one comment.

**Pull request merged**